src/main/java/io/supertokens/ActiveUsers.java

```diff
@@ -11,4 +11,8 @@ public static void updateLastActive(Main main, String userId) {
         } catch (StorageQueryException ignored) {
         }
```

Comment:
- in all the relevant APIs, first send the response, and then call the last update time (first to a test for 20 seconds waiting)
- in session apis, the part where you are calling a user id mapping, it needs to be in a try / catch.